### PR TITLE
feat(internal): allow seed cli to register versions of generators and CLIs

### PR DESCRIPTION
**Pull request by armandobelardo**
Fixes FER-2914
Introduces version files that declare the release types consumed by FDR

This PR seeds examples for the latest versions of the CLI and python-sdk
```
packages/cli/cli/versions.yml
generators/python/sdk/versions.yml
```

### CODE REVIEW

File: .github/workflows/seed.yml
Comment (armandobelardo) [inline .github/workflows/seed.yml:1]: The changes in this file are just adding the call to the `seed register` command once the seed checks have succeeded

File: packages/seed/fern/definition/config.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
 imports:
   features: features.yml
 types:
+  MinimalSeedWorkspaceConfiguration:
```
Comment (armandobelardo) [inline packages/seed/fern/definition/config.yml:4]: This is to allow the CLI to have it's own seed folder and `seed.yml` without any of the other metadata we currently track for seed

Comment (armandobelardo) [reply packages/seed/fern/definition/config.yml:4]: Allowing us to add in additional configuration down the line, as well as specify a location for the changelog within another project

File: packages/seed/src/cli.ts [hunk 2]
```diff
@@ -192,6 +197,60 @@ function addRunCommand(cli: Argv) {
...
+                .option("generator", {
+                    type: "array",
+                    string: true,
+                    demandOption: false,
+                    description: "Generator(s) to register"
+                })
+                .option("cli", {
+                    boolean: true,
+                    default: false,
+                    description: "Initialize a docs website."
```
Comment (dsinghvi) [inline packages/seed/src/cli.ts:215]: is this the right description?

File: packages/seed/fern/definition/config.yml (hunk 1 above)
Comment (dsinghvi) [reply packages/seed/fern/definition/config.yml:4]: Up to call this `CliSeedWorkspaceConfiguration` ? dont foresee us needing to add more so down to make the naming here hyperspecific

**Review by dsinghvi — APPROVED** [review]

**Pull request merged**